Sources/Root/Cells/SearchQueryCell.swift

```diff
@@ -4,25 +4,9 @@
 
 import UIKit
 
-protocol SearchQueryCellDelegate: AnyObject {
-    func searchQueryCellDidTapSearchBar(_ searchQueryCell: SearchQueryCell)
-    func searchQueryCellDidTapRemoveSelectedValue(_ searchQueryCell: SearchQueryCell)
-}
-
 class SearchQueryCell: UITableViewCell {
-    weak var delegate: SearchQueryCellDelegate?
-
-    private lazy var searchBar: UISearchBar = {
-        let searchBar = SearchQueryCellSearchBar(frame: .zero)
-        searchBar.searchBarStyle = .minimal
-        searchBar.delegate = self
-        return searchBar
-    }()
-
-    private var hasTappedClearButton = false
-
-    override var textLabel: UILabel? {
-        return nil
+    var searchBar: UISearchBar? {
+        didSet { didSet(searchBar) }
```

Comment:
I think I would prefer this line to have some newlines 🙂 

And a different name for `didSet` (suggestion: `setupSearchBar`?)

Reply:
👍